src/ariadne.c

```diff
@@ -383,8 +383,18 @@ long Keeper_nav_rulesA2B(long treeA, long treeB)
 
 long navigation_rule_normal(long treeA, long treeB)
 {
-    if ((treeB & 0x0F) - (treeA & 0x0F) > 1)
-      return 0;
+    int difference = ((treeB & 0x0F) - (treeA & 0x0F));
+    if (difference > 1) // Creatures can walk over one block height difference
+    {
+        if (difference < COLUMN_STACK_HEIGHT)
```

Comment:
don't think COLUMN_STACK_HEIGHT makes sense here, the 8 is for the highest possible cubes, like the dungeon hart pillars, those are higher then walls, won't this if always be true?

Reply:
I don't know why not, but I logged the values, and filled slabs had values between 11 and 20 or something, and open slabs up to 6. If we don't think it's cubes, we could change it to simply '8'. Or do you have another idea what it could be?